content/docs/aws/deploy/install-kubeflow.md

```diff
@@ -41,13 +41,13 @@ some handy environment variables:
 
 1. Unpack the tar ball:
 
-    ```
+    ```bash
```

Comment:
Unfortunately, adding the language identifier after the code formatting markers breaks the list formatting on the rendered page:
https://deploy-preview-1723--competent-brattain-de2d6d.netlify.com/docs/aws/deploy/install-kubeflow/

To fix, remove the `bash` annotation on all code snippets that are part of a list.

Reply:
My bad. I noticed this issue before and carelessly forget it this time.